src/Codeception/Module/ZF2.php

```diff
@@ -107,8 +120,28 @@ public function _after(TestCase $test)
     }
 
     public function _getEntityManager()
-    {
-        $serviceLocator = Application::init($this->applicationConfig)->getServiceManager();
-        return $serviceLocator->get('Doctrine\ORM\EntityManager');
+    {   
+        return $this->grabServiceFromContainer('Doctrine\ORM\EntityManager');
```

Comment:
once again, why did you define `entityManager` property if we have this nice method? ;)